service/service.go

```diff
@@ -272,6 +274,54 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var remotewriteController *remotewrite.Controller
+	{
+		if shouldCreateCAPIController(provider) {
```

Comment:
Why is this only for CAPI Managemement cluster ? I do not think this filter is needed

Reply:
Copy-paste stuff 😳